processors/record_linker/processor.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import uuid
+import logging
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+
+    # Trial m2m
+    TRIAL_TABLES = [
```

Comment:
Added a new PR that makes this easier: https://github.com/opentrials/api/pull/73

After that's merged (hopefully before this), you won't need to bother with those relationship tables, but will still have to delete from the `documents` table.